staging/prometheus-operator/templates/prometheus/clusterrole.yaml

```diff
@@ -32,4 +32,18 @@ rules:
   verbs: ["get", "list", "watch"]
 - nonResourceURLs: ["/metrics"]
   verbs: ["get"]
+{{- range .Values.mesosphereResources.additionalPrometheusRBACRules }}
+- apiGroups:
+  {{- range .apiGroups }}
+  - {{ . }}
+  {{- end }}
+  resources:
+  {{- range .resources }}
+  - {{ . }}
+  {{- end }}
+  verbs:
+  {{- range .verbs }}
+  - {{ . }}
+  {{- end }}
+{{- end }}
```

Comment:
think you can do something like:

`{{- toYaml .Values.mesosphereResources.additionalPrometheusRBACRules | nindent 4 }}`

instead of all this code. might need to play around with `nindent` a bit.